packages/skatejs/src/with-initializer.js

```diff
@@ -0,0 +1,13 @@
+// @flow
+
+export const withInitializer = (Base: Class<any> = HTMLElement): Class<any> =>
```

Comment:
I think we should remove the implementation of this in `with-update` and explicitly use this mixin in it to do the props initialisation.